pkg/kf/testutil/integration.go

```diff
@@ -1009,6 +1009,7 @@ func (k *Kf) CreateServiceBroker(ctx context.Context, brokerName string, url str
 		"--namespace", SpaceFromContext(ctx),
 		brokerName,
 		url,
+		"--space-scoped",
```

Comment:
This conflicts with the "--space-scoped" that's in the integration tests. I think this one should be removed.